subprojects/packagefiles/sdl3_ttf/meson.build

```diff
@@ -0,0 +1,77 @@
+project(
+  'sdl3_ttf',
+  'c',
+  license: 'zlib',
+  version: '3.2.2',
+)
+
+version = {
```

Comment:
This should be `meson.project_version().split('.')` rather than hardcoding the version twice.